cpp/include/nvtx3/nvtx3.hpp

```diff
@@ -2113,100 +2113,100 @@ inline void end_range(range_handle r) noexcept
  * When constructed, begins a NVTX range in the specified domain. Upon
  * destruction, ends the NVTX range.
  *
- * Similar to `nvtx3::domain_thread_range`, the only difference being that
- * `domain_process_range` can start and end on different threads.
+ * Similar to `nvtx3::scoped_range_in`, the only difference being that
+ * `unique_range_in` can start and end on different threads.
```

Comment:
I know it's not directly related to the rename change, but I think the comments here aren't totally accurate.  The difference you mention is true, but there are more differences.  Scoped ranges must end in reverse-creation order, following push/pop semantics.  Unique ranges can end in any order.  Unique ranges are also movable, whereas scoped ranges are not.  And unique ranges can be created on the heap, whereas scoped ranges cannot.  So even in single-threaded programs there may be situations where unique_range is a better choice than scoped_range.  We can say that here or in a different PR...  your call.